examples/python/text_categorizer_multiple_models.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python
+#
+#    This example shows that a feature extractor can be shared between two
+#    text categorizers
+#
+import sys, os
+
+# Make sure you put the mitielib folder into the python search path.  There are
+# a lot of ways to do this, here we do it programmatically with the following
+# two statements:
+parent = os.path.dirname(os.path.realpath(__file__))
+sys.path.insert(0, parent + '/../../mitielib')
+from mitie import *
+
+fe_filename= "../../MITIE-models/english/total_word_feature_extractor.dat"
+trainer1 = text_categorizer_trainer(fe_filename)
+
+# Don't forget to add the training data.  Here we have only two examples, but for real
+# uses you need to have thousands. You could also pass whole sentences in to the tokenize() function
+# to get the tokens.
+trainer1.add_labeled_text(["I","am","so","happy","and","exciting","to","make","this"],"positive")
+trainer1.add_labeled_text(["What","a","black","and","bad","day"],"negative")
+
+
+
+# The trainer can take advantage of a multi-core CPU.  So set the number of threads
+# equal to the number of processing cores for maximum training speed.
+trainer1.num_threads = 4
+
+
+# This function does the work of training.  Note that it can take a long time to run
+# when using larger training datasets.  So be patient.
+cat1 = trainer1.train()
+
+# Now that training is done we can save the categorizer object to disk like so.
+# In pure_model mode we do not include a copy of the feature extractor.
+cat1.save_to_disk("new_text_categorizer_1_pure_model.dat",pure_model=True)
+
+trainer2 = text_categorizer_trainer(fe_filename)
+trainer2.add_labeled_text(tokenize("Recharge my phone"),"positive")
+trainer2.add_labeled_text(tokenize("Recharge my number"),"positive")
+trainer2.add_labeled_text(tokenize("I want to recharge my phone"),"positive")
+trainer2.add_labeled_text(tokenize("Cancel"),"negative")
+trainer2.add_labeled_text(tokenize("Finish"),"negative")
+trainer2.add_labeled_text(tokenize("Close"),"negative")
+trainer2.num_threads = 4
+cat2 = trainer2.train()
+cat2.save_to_disk("new_text_categorizer_2_pure_model.dat",pure_model=True)
+
+
+text_feature_extractor = total_word_feature_extractor(fe_filename)
+
+cat1_new = text_categorizer("new_text_categorizer_1_pure_model.dat", None, True)
```

Comment:
The user shouldn't have to supply None,True as an argument.  The code can figure out that the file on disk is a pure model so it should not require the user to supply this redundant information.